### PR TITLE
Debug a node app using Chrome DevTools

### PR DESCRIPTION
**Pull request by geoffrey45**
I have written an article on debugging a Node.js app using Chrome DevTools.

I have covered how to set up a local and remote app for debugging. How to set up an SSH tunnel to debug a remote app. Opening DevTools, setting breakpoints, and restarting app instance for debugging. Using curl to restart a debug session.

### CODE REVIEW

**Comment by hectorkambow** [comment]
Plagiarism check DONE

**Comment by geoffrey45** [comment]
i'm getting into it

**Comment by geoffrey45** [comment]
Hello @sudosoph. I have made the edits.

File: articles/debug-node-devtools/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+The most popular way JavaScript and Node developers track bugs in their programs is printing data to the console. They use methods like [console.log()](https://developer.mozilla.org/en-US/docs/Web/API/Console/log) and [console.debug()](https://developer.mozilla.org/en-US/docs/Web/API/Console/debug). Although these methods can be helpful, they require a lot of manual effort and are not foolproof. Suppose you have a large app, how many `console.log()` statements would you need? Hundreds, perhaps. 
+
+Logging sensitive information can occur, exposing your app to malicious exploitations and data breaches. Debuggers come handy in a situation like this. They help track bugs without exposing your app to security threats.
+
+Sometimes Node.js apps can fail without error messages. This occurs mainly in cases where [error handling](https://nodejs.dev/learn/error-handling-in-nodejs) is not employed. Debuggers can watch behaviors of specific objects throughout program execution by adding breakpoints in a program. Breakpoints are points in your code where you pause execution and examine various factors. By watching objects, you can evaluate their values and type as the execution goes on.
```
Comment (sudosoph) [inline articles/debug-node-devtools/index.md:29]: Even so, Node.js apps can fail without error messages.

Comment (sudosoph) [reply articles/debug-node-devtools/index.md:29]: Breakpoints are locations in your code where you pause execution and examine various factors.

File: articles/debug-node-devtools/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+```bash
+Debugger listening on ws://127.0.0.1:9229/31dc8b93-3f2c-4352-aa7b-e9357bbabccc
+For help, see: https://nodejs.org/en/docs/inspector
+Listening on http://127.0.0.1:8000
+```
+
+### Debugging a remote app 
+
+If you want to debug your remote in your browser, it's recommended to use SSH tunnels. By doing this, I assume that you understand the security risks associated with allowing remote access to privileged services. Unauthorized database access, website defacement and phishing attacks are some of these risks.
```
Comment (sudosoph) [inline articles/debug-node-devtools/index.md:56]: If you want to debug your remote in your browser, it's best to use SSH tunnels.

Comment (sudosoph) [reply articles/debug-node-devtools/index.md:56]: However, there are security risks associated with allowing remote access to privileged services. These include unauthorized database access, website defacement and phishing attacks.

**Comment by hectorkambow** [comment]
Good afternoon @jstangroome  👋 I know this article isn't exactly security but I saw some security aspects in the article that I thought would benefit from your eye and input.
If you could give this _Debugging a Node.js app using Devtools_ a tech review whenever you get a moment.
We've recently started running our EngEd articles through our Peer Reviewer Team - and the process should be a lot smoother moving forward. 
Just wanted to make sure the information here is accurate and we didn't miss anything.
Thank you again 🚀

**Review by jstangroome — APPROVED** [review]

**Pull request merged**